--- /home/z/.zshrc.orig
+++ /home/z/.zshrc
@@ -7,7 +7,7 @@
 # Set name of the theme to load. Optionally, if you set this to "random"
 # it'll load a random theme each time that oh-my-zsh is loaded.
 # See https://github.com/robbyrussell/oh-my-zsh/wiki/Themes
-ZSH_THEME="robbyrussell"
+ZSH_THEME=""
 
 # Set list of themes to load
 # Setting this variable when ZSH_THEME=random
@@ -60,6 +60,10 @@
 # Add wisely, as too many plugins slow down shell startup.
 plugins=(
   git
+  z
+  colored-man-pages
+  zsh-autosuggestions
+  zsh-syntax-highlighting
 )
 
 source $ZSH/oh-my-zsh.sh
@@ -92,3 +96,17 @@
 # Example aliases
 # alias zshconfig="mate ~/.zshrc"
 # alias ohmyzsh="mate ~/.oh-my-zsh"
+#alias grep='grep --color=auto'
+#alias ls='ls --color=auto'
+alias l='ls -alF'
+alias la='ls -aF'
+alias ll='ls -lF'
+
+unsetopt share_history
+
+# My Prompt, derived from the default robbyrussell theme
+PROMPT='%n@%m%{$fg[magenta]%}:%{$reset_color%}%~$(git_prompt_info)%(?:%#:%{$fg[red]%}%#)%{$reset_color%} '
+ZSH_THEME_GIT_PROMPT_PREFIX="%{$fg[blue]%} (%{$fg[cyan]%}"
+ZSH_THEME_GIT_PROMPT_SUFFIX="%{$reset_color%}"
+ZSH_THEME_GIT_PROMPT_DIRTY="%{$fg[yellow]%}*%{$fg[blue]%})"
+ZSH_THEME_GIT_PROMPT_CLEAN="%{$fg[blue]%})"
