--- /home/z/.zshrc.orig
+++ /home/z/.zshrc
@@ -60,6 +60,10 @@
 # Add wisely, as too many plugins slow down shell startup.
 plugins=(
   git
+  z
+  colored-man-pages
+  zsh-autosuggestions
+  zsh-syntax-highlighting
 )
 
 source $ZSH/oh-my-zsh.sh
@@ -92,3 +96,9 @@
 # Example aliases
 # alias zshconfig="mate ~/.zshrc"
 # alias ohmyzsh="mate ~/.oh-my-zsh"
+#alias grep='grep --color=auto'
+#alias ls='ls --color=auto'
+alias l='ls -alF'
+alias la='ls -aF'
+alias ll='ls -lF'
+unsetopt share_history
